src/Sarif.UnitTests/JsonTests.cs

```diff
@@ -13,7 +13,7 @@ namespace Microsoft.CodeAnalysis.Sarif
     public abstract class JsonTests
     {
         protected static readonly Run DefaultRun = new Run();
-        protected static readonly Tool DefaultTool = new Tool();
+        protected static readonly Tool DefaultTool = new Tool() { Name = "DefaultTool" };
```

Comment:
>Name = "DefaultTool"  [](start = 66, length = 21)

With the jschema 0.61.0 update, the validator now enforces the "required" element. And so there are several updates where tests did not previously provided required values.